Test.Integration.Net7/EventsAPI/Account.cs

```diff
@@ -0,0 +1,199 @@
+using Sift;
+using System.Collections.ObjectModel;
+using Test.Integration.Net7.Uitlities;
+using Xunit;
+
+namespace Test.Integration.Net7.EventsAPI
+{
+    public class Account
+    {
+        private readonly EnvironmentVariable environmentVariable = new();
+        [Fact]
+        public void IntegrationTest_CreateAccount()
+        {
+            var sift = new Client(environmentVariable.ApiKey);
+            var createAccount = new CreateAccount
+            {
+                user_id = environmentVariable.user_id,
```

Comment:
we don't need to read `user_id`, `session_id`, `user_email` etc from env variables, it is ok to hard code them here. The only things we would like to move to the env variables are `api_key` and `account_id`

Reply:
@binish-exalture pls, take a look